src/components/menu-left/MenuLeft.vue

```diff
@@ -1,14 +1,68 @@
 <template>
   <div class="column is-narrow menu-left">
     <h3>Menu left</h3>
+    <section>
```

Comment:
`section` and `footer` are good for semantics in websites, but for web applications they sometimes add unnecessary default styling. We should use `div` instead.